CHANGELOG.md

```diff
@@ -1,8 +1,115 @@
 # 2.0.0 ()
 
+BREAKING CHANGES:
+
+Cluster
+* **Rename Resource:** `netapp-ontap_cluster_licensing_license_resource` is now renamed to `netapp-ontap_cluster_licensing_license`
+* **Rename Resource:** `netapp-ontap_cluster_peers_resource` is now renamed to `netapp-ontap_cluster_peer`
+* **Rename Resource:** `netapp-ontap_cluster_schedule_resource` is now renamed to `netapp-ontap_cluster_schedule`
+* **Rename Data Source:** `netapp-ontap_cluster_licensing_license_data_source` is now renamed to `netapp-ontap_cluster_licensing_license`
+* **Rename Data Source:** `netapp-ontap_cluster_licensing_licenses_data_source` is now renamed to `netapp-ontap_cluster_licensing_licenses`
+* **Rename Data Source:** `netapp-ontap_cluster_peer_data_source` is now renamed to `netapp-ontap_cluster_peer`
+* **Rename Data Source:** `netapp-ontap_cluster_peers_data_source` is now renamed to `netapp-ontap_cluster_peers`
+* **Rename Data Source:** `netapp-ontap_cluster_schedule_data_source` is now renamed to `netapp-ontap_cluster_schedule`
+* **Rename Data Source:** `netapp-ontap_cluster_schedules_data_source` is now renamed to `netapp-ontap_cluster_schedules`
+
+Name Services
+* **Rename Resource:** `netapp-ontap_name_services_dns_resource` is now renamed to `netapp-ontap_dns`
+* **Rename Resource:** `netapp-ontap_name_services_ldap_resource` is now renamed to `netapp-ontap_name_services_ldap`
+* **Rename Data Source:** `netapp-ontap_name_services_dns_data_source` is now renamed to `netapp-ontap_dns`
+* **Rename Data Source:** `netapp-ontap_name_services_dnss_data_source` is now renamed to `netapp-ontap_dnss`
+* **Rename Data Source:** `netapp-ontap_name_services_ldap_data_source` is now renamed to `netapp-ontap_name_services_ldap`
+* **Rename Data Source:** `netapp-ontap_name_services_ldaps_data_source` is now renamed to `netapp-ontap_name_services_ldaps`
+
+Networking
+* **Rename Resource:** `netapp-ontap_networking_ip_interface_resource` is now renamed to `netapp-ontap_network_ip_interface`
+* **Rename Resource:** `netapp-ontap_networking_ip_route_resource` is now renamed to `netapp-ontap_network_ip_route`
+* **Rename Data Source:** `netapp-ontap_networking_ip_interface_data_source` is now renamed to `netapp-ontap_network_ip_interface`
+* **Rename Data Source:** `netapp-ontap_networking_ip_interfaces_data_source` is now renamed to `netapp-ontap_network_ip_interfaces`
+* **Rename Data Source:** `netapp-ontap_networking_ip_route_data_source` is now renamed to `netapp-ontap_network_ip_route`
+* **Rename Data Source:** `netapp-ontap_networking_ip_routes_data_source` is now renamed to `netapp-ontap_network_ip_routes`
+
+Protocols
+* **Rename Resource:** `netapp-ontap_cifs_local_group_member_resource` is now renamed to `netapp-ontap_cifs_local_group_members`
```

Comment:
Should "member" or "members"?

Reply:
"members", i check the API, you can create a group of members at one time. We haven't implemented it, but it is there.